jfrog-cli/docs/artifactory/curl/help.go

```diff
@@ -0,0 +1,8 @@
+package curl
+
+const Description = "Run Curl commands to use Artifactory REST API."
```

Comment:
Run Curl commands to use Artifactory REST API. --> Execute s cUrl command, using the configured Artifactory details.